lib/tic_tac_toe.rb

```diff
@@ -0,0 +1,117 @@
+require_relative 'players'
+# Tic-Tac-Toe executable file
+class TicTacToe
+  WINNING_COMBINATION = [
+    [0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6],
+    [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]
+  ].freeze
+
+  def initialize
+    @board = %w[_ _ _ _ _ _ _ _ _]
```

Comment:
Do you need to manually store `_`? This is error-prone. One typo and this breaks.

Reply:
Also hard to debug